tests/test_clearmash_api.py

```diff
@@ -154,3 +158,19 @@ def test_get_documents_places():
+
+def test_get_related_docs_of_places_item():
+    try:
+        documents_response = MockClearmashApi().get_document_photos_docs(220590)
+        # comapre related elemts with response for api.get_documents(["220590"])
+        reference_data = GERMANY_FROM_GET_DOCUMENTS_CLEARMASH_API
+        entities = documents_response.pop("Entities")
+        related_documents = reference_data["Entities"][0]["Document"]["Fields_RelatedDocuments"][0]
+        for i in related_documents:
+            if i["Id"] == "_c6_beit_hatfutsot_bh_base_template_multimedia_photos":
```

Comment:
this is a lot of logic, which will have to be done every time to "get_document_photos_docs" - which is why it would be better to change it to a simple interface over clearmash - as I wrote in above comment